doc/reference/ufunc.rst

```diff
@@ -5,7 +5,49 @@ Universal Functions (ufunc)
 
 .. https://docs.scipy.org/doc/numpy/reference/ufuncs.html
 
-DPNP provides universal functions (a.k.a. ufuncs) to support various element-wise operations.
+A universal function (or ufunc for short) is a function that operates on ndarrays in an element-by-element fashion, \
+supporting array broadcasting, type casting, and several other standard features. That is, a ufunc is a “vectorized” \
+wrapper for a function that takes a fixed number of specific inputs and produces a fixed number of specific outputs. \
+For full documentation refer to :obj:`numpy.ufunc`.
+
+ufuncs
+------
+.. autosummary::
+   :toctree: generated/
+
+   dpnp.ufunc
+
+Attributes
+~~~~~~~~~~
+
```

Comment:
Do we need to specify `index` directive? (applicable to different place on the page)
```rst
.. index::
   pair: ufunc; attributes
```